jenkins.sh

```diff
@@ -14,5 +14,8 @@ if [ ! -f .ruby-version ]; then
   export PATH=$(printf $PATH | awk 'BEGIN { RS=":"; ORS=":" } !/rbenv/' | sed 's/:$//')
 fi
 
+# Reset the RestClient log to an empty file
+cat /dev/null >log/smokey-rest-client.log
```

Comment:
Instead of this, it's probably easiest to do a `git clean -fdx` at the top of the file.  This is what we do for most other projects.